spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -190,4 +238,35 @@ public void onChangeEvent(ConfigurationChangeEvent event) {
             disable = Boolean.parseBoolean(event.getNewValue().trim());
         }
     }
+
+    /**
+     * auto upgrade service detection
+     */
+    private static void startSelfCheck() {
+        executor.scheduleAtFixedRate(() -> {
+            if (demotionMap.size() > 0) {
+                try {
+                    TransactionManagerHolder.get()
```

Comment:
split

Reply:
split begin and commit, will be more clearly.

Reply:
> split begin and commit, will be more clearly.

i modified it